app/views/layouts/application.html.erb

```diff
@@ -2,6 +2,7 @@
 <html>
   <head>
     <%= render "layouts/google_analytics" %>
```

Comment:
What's the plan for google analytics once segment is live? Is there a task to replace it?

Reply:
I think we could get rid of Google Analytics eventually esp. if we forward Segment events there. Made a ticket for it https://jira.czi.team/browse/IDSEQ-841